esptfa_arima/Authentication/views.py

```
@@ -122,15 +175,22 @@ def me(self, request):
-            token = RefreshToken(refresh_token)
-            token.blacklist()
-            return Response({"detail": "Successfully logged out"}, status=status.HTTP_200_OK)
+            refresh_token = request.COOKIES.get(settings.SIMPLE_JWT['AUTH_COOKIE_REFRESH'])
+            if refresh_token:
+                token = RefreshToken(refresh_token)
+                token.blacklist()
+            
+            response = Response({"detail": "Successfully logged out"}, status=status.HTTP_200_OK)
+            response.delete_cookie(settings.SIMPLE_JWT['AUTH_COOKIE'], path='/')
+            response.delete_cookie(settings.SIMPLE_JWT['AUTH_COOKIE_REFRESH'], path='/')
+            return response
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)